From f5617144fc5a76140016348c87a24adcd3fe2656 Mon Sep 17 00:00:00 2001
From: David Reaver <me@davidreaver.com>
Date: Tue, 28 Jan 2025 19:16:25 -0800
Subject: [PATCH 0/1] *** SUBJECT HERE ***
To: Steven Rostedt <rostedt@goodmis.org>,
    Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Tejun Heo <tj@kernel.org>,
    Christian Brauner <brauner@kernel.org>,
    linux-fsdevel@vger.kernel.org,
    linux-kernel@vger.kernel.org

*** BLURB HERE ***

David Reaver (1):
  debugfs: Use opaque debugfs_node handle to support kernfs migration

 arch/powerpc/kernel/kdebugfs.c |   2 +-
 arch/s390/kernel/kdebugfs.c    |   2 +-
 arch/sh/kernel/kdebugfs.c      |   2 +-
 arch/x86/kernel/cpu/debugfs.c  |   2 +-
 arch/x86/kernel/kdebugfs.c     |  12 +--
 block/blk-core.c               |   6 +-
 block/blk.h                    |   2 +-
 drivers/acpi/debugfs.c         |   2 +-
 drivers/acpi/internal.h        |   2 +-
 drivers/base/component.c       |   2 +-
 drivers/char/virtio_console.c  |   4 +-
 drivers/dma/dmaengine.c        |   2 +-
 drivers/dma/dmaengine.h        |   2 +-
 drivers/gpu/drm/drm_drv.c      |   2 +-
 drivers/misc/mei/mei_dev.h     |   2 +-
 fs/debugfs/file.c              | 192 ++++++++++++++++++---------------
 fs/debugfs/inode.c             | 126 ++++++++++++----------
 include/linux/blkdev.h         |   6 +-
 include/linux/debugfs.h        | 175 +++++++++++++++---------------
 include/linux/dmaengine.h      |   2 +-
 include/net/cfg80211.h         |   2 +-
 mm/vmstat.c                    |   2 +-
 net/wireless/core.c            |   2 +-
 net/wireless/debugfs.c         |   2 +-
 24 files changed, 298 insertions(+), 257 deletions(-)


base-commit: 05dbaf8dd8bf537d4b4eb3115ab42a5fb40ff1f5
